<!-- <?xml version="1.0" ?> -->
<!-- <robot name="sawyer" xmlns:xacro="http://www.ros.org/wiki/xacro"> -->
<launch>


    <group ns="ref_sawyer">
        <param name="robot_description" command="$(find xacro)/xacro  --inorder '$(find sawyer_description)/urdf/sawyer.urdf.xacro'" />
        <param name="tf_prefix" type="string" value="ref_tf"/>
        <arg name="robot_name" value="ref_bot"/>
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher"/>
        <node name="ref_js_randomizer" pkg="sawyer_velctrlsim" type="ref_rand_joint_state_pub.py"/>
    </group>

    <group ns="main_sawyer">
        <param name="robot_description" command="$(find xacro)/xacro  --inorder '$(find sawyer_description)/urdf/sawyer.urdf.xacro'" />
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
            <param name="tf_prefix" type="string" value="main_tf"/>
        </node>
        <node name="ref_vel_ctrl" pkg="sawyer_velctrlsim" type="ref_vel_ctrl.py" output="screen"/>
    </group>

    <node pkg="tf" type="static_transform_publisher" name="fixed_frame_pub" args="0 0 0 0 0 0 ref_tf/base main_tf/base 100"/>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find sawyer_velctrlsim)/rviz/velctrl.rviz" required="true"/>

</launch>
